api/defaults/defaults.go

```diff
@@ -139,6 +139,11 @@ func EnhancedEvents() []string {
 	}
 }
 
+const (
+	// MaxIterationLimit is max iteration limit
```

Comment:
I'd probably just use a private const for this. It's too generic as-is to put here.

Reply:
unexported and changed `lib/defaults` to also have a value of `1000`.